web/cobrands/sass/_admin.scss

```diff
@@ -323,3 +323,8 @@ $button_bg_col: #a1a1a1;  // also search bar (tables)
     max-height: initial;
   }
 }
+
+textarea.code {
+    font-family: monospace;
+    max-width: initial;
+}
```

Comment:
Github has a warning(?) about no newline at the end of the file - is this something we are concerned about?

Reply:
Yep, always good to get rid of warnings! Have fixed.